src/api.rs

```diff
@@ -10,6 +10,34 @@ use crate::helpers;
 use crate::responses::*;
 use storage_proofs::sector::SectorId;
 
+#[cfg(not(target_os = "windows"))]
+pub unsafe fn raw_to_file(raw: *mut libc::c_void) -> std::fs::File {
```

Comment:
BLOCKING: 

Two things:

1. This should be `libc::c_int`, not `*mut libc::c_void`.
1. You should extract these functions into `rust-fil-ffi-toolkit`.

Reply:
I can’t use different types, between windows and others because I cannot get cbindgen to generate different versions depending target os, so I settled on the generic *mut void for both

Reply:
> I can’t use different types

Sorry, I think I miscommunication. I propose that both Windows and *nix versions of `raw_to_file` have the following signature:

```
pub unsafe fn raw_to_file(raw: libc::c_int) -> std::fs::File {
```

I've made the change locally and things compile and run you'd expect. cbindgen produces a header file with the following prototype:

```
sector_builder_ffi_AddPieceResponse *sector_builder_ffi_add_piece(sector_builder_ffi_SectorBuilder *ptr,
                                                                  const char *piece_key,
                                                                  int piece_fd,
                                                                  uint64_t piece_bytes_amount,
                                                                  uint64_t store_until_utc_secs);
```